common/src/service/IotHubConnectionString.cs

```diff
@@ -108,16 +86,6 @@ Task<CbsToken> ICbsTokenProvider.GetTokenAsync(Uri namespaceAddress, string appl
             return Task.FromResult(token);
         }
 
-        public Uri BuildLinkAddress(string path)
```

Comment:
public methods in internal classes always gets me confused! :D

Reply:
The point is that you can always just change the access modifier of the class to easily make things public and private and internal. But this specific method can be private in the base class.

Reply:
Actually I take it back. I tried to do that and remembered why it did not work. It is used in a public method and there is a chain so it does not work.